src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -216,6 +224,27 @@ public boolean hasBid(Bid bid) {
         return bidBook.hasBid(bid);
     }
 
+    @Override
+    public void isValidBid(Bid bid) throws CommandException {
```

Comment:
Maybe should change to `checkIsValidBid` because this name suggests that the method will return a boolean. Alternatively, you can also make this return true if no exception is thrown.